edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Edgelet/ModuleIdentityLifecycleManager.cs

```diff
@@ -100,12 +129,18 @@ static class Events
             enum EventIds
             {
                 ErrorGettingModuleIdentities = IdStart,
+                RemoveOrphanedIdentities,
             }
 
             public static void ErrorGettingModuleIdentities(Exception ex)
             {
                 Log.LogDebug((int)EventIds.ErrorGettingModuleIdentities, ex, "Error getting module identities.");
             }
+
+            public static void RemoveOrphanedIdentities(IEnumerable<string> removeOrphanedIdentities)
+            {
+                Log.LogDebug((int)EventIds.RemoveOrphanedIdentities, $"Removing orphaned identities {string.Join(", ", removeOrphanedIdentities.Select(s => s.ToString()))}");
```

Comment:
I think this should be Info just to make sure if it happens that we removed host identities and someone reports it. We should not see these logs too often so it should be ok to be in Info

Reply:
Sounds good, will promote to info